miso-web/src/main/webapp/pages/editRun.jsp

```diff
@@ -256,9 +256,22 @@
         </thead>
         <tbody>
         <tr>
-          <td><fmt:formatDate pattern="dd/MM/yyyy" value="${run.startDate}"/></td>
           <c:choose>
-          <c:when test="${(run.health.isDone() and empty run.completionDate)}">
+          <c:when test="${run.id == 0 or fn:contains(SPRING_SECURITY_CONTEXT.authentication.principal.authorities,'ROLE_ADMIN')}">
+              <td><form:input path="startDate"/></td>
+          
+              <script type="text/javascript">
+              Utils.ui.addDatePicker("startDate");
+              </script>
+            </c:when>
+            <c:otherwise>
+              <td id="startDate">
+                <fmt:formatDate pattern="dd/MM/yyyy" value="${run.startDate}"/>
+              </td>
+            </c:otherwise>
+          </c:choose>
+          <c:choose>
+          <c:when test="${(run.health.isDone() and empty run.completionDate) or run.id == 0 or fn:contains(SPRING_SECURITY_CONTEXT.authentication.principal.authorities,'ROLE_ADMIN')}">
```

Comment:
True... Also as is, if you wanted to mark something completed and add the completionDate at the same time (as would be normal), you would have to
1. change to Complete
2. save
3. set completionDate
4. save again